packages/manager/test/__testutils__/createAPIFixture.ts

```diff
@@ -0,0 +1,101 @@
+import { SetupServer } from "msw/lib/node";
+import { rest } from "msw";
+
+import { SliceMachineManager } from "../../src";
+
+type MockOptions = {
+	method?: keyof typeof rest;
+	statusCode?: number;
+	checkAuthentication?: boolean;
+	searchParams?: Record<string, string>;
+	body?: SerializableValue;
+};
+
+export type SerializableValueObject = { [Key in string]: SerializableValue } & {
+	[Key in string]?: SerializableValue | undefined;
+};
+export type SerializableValueArray =
+	| SerializableValue[]
+	| readonly SerializableValue[];
+export type SerializableValuePrimitive =
+	| string
+	| number
+	| boolean
+	| Date
+	| null;
+export type SerializableValue =
```

Comment:
I prefer exporting any top-level types that a function uses since those types may want to be referenced by a consumer. For example, `SerializableValue` may be used to declare a constant to be passed to the function call:

```ts
const fixture = createAPIFixture({ manager, mswServer });

const response = { foo: "bar" } satisfies SerializableValue;
fixture.mockSliceMachineV1("./git/owners", response);
```

The above example is contrived, but this practice can be useful in more specific cases.

When a type is not exported, consumers are forced to extract a type from another type, sometimes via `typeof` or without the ability to use generics:

```ts
const fixture = createAPIFixture({ manager, mswServer });

const response = { foo: "bar" } satisfies Parameters<typeof fixture['mockSliceMachineV1']>[1];
fixture.mockSliceMachineV1("./git/owners", response);
```

That being said, there is no need to export all of `SerializableValue`'s child types, like `SerializableValuePrimitive`.